5.4.connected_vessels.py

```diff
@@ -0,0 +1,37 @@
+# The function accepts one or more iterable parameters, and returns a list of the interlaced elements
+def interleave(*iterable):
+    if not iterable:
+        return None
+    res = []
+    j = 0
```

Comment:
Try to avoid i and j as well if there is a better name available.

Reply:
I fixed it.